app/scripts/modules/core/src/search/global/GlobalSearch.tsx

```diff
@@ -1,27 +1,30 @@
 import * as React from 'react';
+import { UIRouterContext } from '@uirouter/react-hybrid';
+
 import { BindAll, Debounce } from 'lodash-decorators';
-import { flatten, range } from 'lodash';
+import { flatten } from 'lodash';
 import * as ReactGA from 'react-ga';
-import { UISref } from '@uirouter/react';
-import { UIRouterContext } from '@uirouter/react-hybrid';
+import { Observable } from 'rxjs/Observable';
+import { Subject } from 'rxjs/Subject';
```

Comment:
👍 I pulled this straight out of the `RecentlyViewedItems` component, and assumed it was an intentional attempt to reduce bundle sizes by letting webpack tree-shake rather than pulling in the entire rxjs bundle (which we did with `lodash` on my last project). Will consolidate.

Reply:
we had problems with deep imports (something to do with publishing `@spinnaker/core` as a library and consuming it in deck-nflx).

It's been easiest to consistently import from `rxjs`.  Once we have a mature publish-as-library story, we should revisit how we import from rxjs.